.github/workflows/configure_core.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# You can use your interpreter of choice (bash, sh, zsh, ...)
+
+# For any question please contact with us in:
+#   - https://github.com/KratosMultiphysics/Kratos
+
+# Optional parameters:
+# You can find a list with all the compilation options in INSTALL.md or here:
+#   - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options
+
+add_app () {
+    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
+}
+
+# Set variables
+export KRATOS_SOURCE="${KRATOS_SOURCE:-${PWD}}"
+export KRATOS_BUILD="${KRATOS_SOURCE}/build"
+export KRATOS_APP_DIR="${KRATOS_SOURCE}/applications"
+export PYTHON_EXECUTABLE="/usr/bin/python3.10"
+export KRATOS_INSTALL_PYTHON_USING_LINKS=ON
+
+# Set applications to compile
+add_app ${KRATOS_APP_DIR}/LinearSolversApplication;
+add_app ${KRATOS_APP_DIR}/MetisApplication;
+add_app ${KRATOS_APP_DIR}/TrilinosApplication;
```

Comment:
Yes.. I agree, but for reasons I need those three to be compiled in the first stage no matter what, otherwise the list of dependencies and when to activate them becomes a mess. I may change the name of the script to relfect that :S

(bascially moving trilinos and linear solvers retriggers the compilation of the core no matter what, so ....)